docs/tutorials/tutorial2.md

```diff
@@ -165,6 +165,18 @@ value.
 
     `_addressLine1 = "address1"`
 
+When renaming the name of a field referring to a foreign key (for example
+the `_addressForUser` field), remember to specify the field name as a
+`PrimaryKey` field like:
+
+`_addressForUser = UserId "user"`
```

Comment:
I think this will render better as either a code block or inline with the previous paragraph. (I have a slight preference for the latter.)